gcp/api/server.py

```diff
@@ -99,7 +99,7 @@ def GetVulnById(self, request, context: grpc.ServicerContext):
       context.abort(grpc.StatusCode.PERMISSION_DENIED, 'Permission denied.')
       return None
 
-    return bug_to_response(bug)
+    return bug.to_vulnerability(include_source=True)
```

Comment:
And this can still use the bug to response func with the change to allow include aliases arg to be passed in